SLRemoting/SLObject.m

```diff
@@ -229,7 +229,7 @@ + (NSDictionary *)convertArguments:(NSDictionary *)prop {
         id value = [prop objectForKey:key];
         if ([value isKindOfClass:[NSDate class]]) {
             NSString *jsonString = [value convertToJSONString];
-            value = @{ @"$type": @"date", @"$data": jsonString };
+            value = jsonString ;
```

Comment:
I see. Could you please check what happens when you use the new simplified encoding `testDate?date=1970-01-01T00%3A02%3A03.000Z` when calling a LoopBack 2.x server? Will the server correctly convert the value to a Date instance?

I think this is a subtle but also an important breaking change, could you please add a note to both Release Notes and Migration Guide for 3.0?